--- meson.build	2026-06-02 15:37:00.066523157 +0200
+++ meson.build	2026-06-02 15:37:19.509526815 +0200
@@ -89,13 +89,20 @@
   dependency('threads'),
 ]
 
-adslib = static_library('AdsLib',
+adslib_static = static_library('AdsLib',
   [common_files, router_files],
   include_directories: inc,
   install: true,
 )
 
-install_libs = [ adslib ]
+adslib = shared_library('AdsLib',
+  [common_files, router_files],
+  include_directories: inc,
+  dependencies: libs,
+  install: true,
+)
+
+install_libs = [ adslib_static, adslib ]
 
 adslib_dep = declare_dependency(
   include_directories : inc,
@@ -133,6 +140,8 @@
   include_directories: inc,
   dependencies: libs,
   link_with: adslib,
+  install: true,
+  install_dir: get_option('bindir'),
 )
 
 if get_option('tcadsdll_include') != ''
@@ -163,6 +173,8 @@
       tcadsdll_deps
     ],
     link_with: tcadslib,
+    install: true,
+    install_dir: get_option('bindir'),
   )
 
   tcadstest = executable('tcAdsTest',
