--- meson.build	2026-06-02 16:30:21.126289024 +0200
+++ meson.build	2026-06-02 16:30:21.156840857 +0200
@@ -144,6 +144,8 @@
   include_directories: inc,
   dependencies: libs,
   link_with: adslib,
+  install: true,
+  install_dir: get_option('bindir'),
 )
 
 if get_option('tcadsdll_include') != ''
@@ -174,6 +176,8 @@
       tcadsdll_deps
     ],
     link_with: tcadslib,
+    install: true,
+    install_dir: get_option('bindir'),
   )
 
   tcadstest = executable('tcAdsTest',
